docs/src/getting-started.md

```diff
@@ -1,9 +1,12 @@
 # Getting started
 
-Kani is an open-source verification tool that uses automated reasoning to analyze Rust programs.
+Kani is an open-source verification tool that uses model checking to analyze Rust programs.
```

Comment:
Suggestion: I think it would be nice to add a link (reference) here for model checking. Imagine a developer that has never really try or heard of it before. I don't have any great suggestions on what we could point here, but perhaps something even more general like this [article](https://www.amazon.science/publications/model-checking-as-a-human-endeavor).

Reply:
+1

Reply:
I like the article idea, and that is a great article to mention here, but I would also suggest something more basic that explains what model checking is.

Reply:
I couldn't find a good resource that is basic enough. I recall we've discussed having a section that explains the basics of model checking and other automated reasoning technologies in the past, but I couldn't find an issue related to it. Maybe https://github.com/model-checking/kani/issues/1562?

Reply:
Why not the [wikipedia page](https://en.wikipedia.org/wiki/Model_checking)?

Reply:
We do have the [gentle introduction to automated reasoning](https://www.amazon.science/blog/a-gentle-introduction-to-automated-reasoning), which to me as a beginner, is easier to read and understand than the wikipedia page.

Reply:
I agree with @adpaco-aws too that we should add a page about the basics of model checking in the future.

Reply:
I instead opted to link to our own `tool-comparison.md`.